src/core/client/stream/components/Permalink/PermalinkPopover.tsx

```diff
@@ -0,0 +1,63 @@
+import { Localized } from "fluent-react/compat";
+import React, { CSSProperties } from "react";
+import CopyToClipboard from "react-copy-to-clipboard";
+import { RefHandler } from "react-popper";
+import { Button, ClickOutside, Flex, TextField } from "talk-ui/components";
+import * as styles from "./PermalinkPopover.css";
+
+interface InnerProps {
```

Comment:
Call it `PermalinkPopoverProps`? We usually used `InnerProps` when we use HOCs but we don't here.